lib/views/item/item-detail.js

```diff
@@ -180,12 +184,60 @@ class ItemDetail extends Component {
       <View style={styles.distance}>
         <Image
           style={styles.distanceImage}
-          source={require('../../images/item-scene-room.png')}/>
+          source={{
+            uri: 'ic_place',
```

Comment:
It's not implemented in Mapbox yet. Plus I found working with image assets is really smooth both on iOS and Android. Using Material Design icons, I just had to slide the folders in relevant places.
And of course this support multi screen densities which is good.